helm/rook-operator/charts/rook-ceph-cluster/templates/namespace.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if .Values.global.rookEnabled }}
+{{- if ne .Values.global.cephClusterNamespace .Values.operatorNamespace }}
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: {{ .Values.global.cephClusterNamespace }}
```

Comment:
Will helm fail if this namespace already exists?

Reply:
good point - i've removed management of the namespace and i'll add an explicit instruction to the docs that this needs to be manually created.